statsforecast.php

```diff
@@ -52,6 +53,11 @@ public function __construct()
 		$this->displayName = $this->l('Stats Dashboard');
 		$this->description = $this->l('This is the main module for the Stats dashboard. It displays a summary of all your current statistics.');
 		$this->ps_versions_compliancy = array('min' => '1.6', 'max' => _PS_VERSION_);
+
+		$this->employeeFilters = $this->context->cookie;
+        if (version_compare(_PS_VERSION_, '1.7.0', '>=')) {
```

Comment:
Don't mix tabs and spaces please :)

Reply:
Ping @kelu95

Reply:
This PR was created 5 years ago, I think you can remove it

Reply:
@kelu95 Have you got always the problem ? If yes, we can work with you on this PR, else you can close it. Thanks (and sorry for the delay, we try to improve us every day).

Reply:
I understand, no problem. Haven't use PS since 2015, so I close it